assets/themes/ansi.tmTheme

```diff
@@ -423,6 +423,17 @@
                     <string>#03000000</string>
                 </dict>
             </dict>
+            <dict>
+                <key>name</key>
+                <string>Line highlight style</string>
+                <key>scope</key>
+                <string>line_highlight</string>
```

Comment:
I'm not sure how I feel about this being a scope selector with specific meaning. It's a very clever solution, for sure :) but I wonder if it would be better to re-use/re-purpose a global color scheme "setting" which already exists, like `tags_options` (http://www.sublimetext.com/docs/color_schemes.html#global_settings-tags) which otherwise wouldn't have any use in a tool like `bat`. To me this would seem more portable and unsurprising.

Reply:
@keith-hall wouldn't re-purposing another scope cause more surprise and confusion for people? Are there actually _standard_ scopes at all? My sense is that they are kind of arbitrary, and different editors can have their own, but I might be mistaken.

Reply:
From the [docs](http://www.sublimetext.com/docs/color_schemes.html#global_settings-tags):
> Author of syntaxes can assign whatever scopes they want to a given token. This combined with the fact that there are hundreds of community-maintained syntaxes means that is can be hard to know what scopes to target. The official Scope Naming guidelines were established to help syntax and color scheme authors use a common set, for better interoperability. The Usage in Color Schemes section provides a baseline set of scopes that color scheme authors should strive to handle. 

My understanding is that themes should try to handle the "baseline" scopes, but they are still free to add their own. But again, I'm new to this area and I'm sure you know better about the best practices on this subject.

Reply:
It's a good point - as I understand, syntax definitions define the scopes for the tokens, and color schemes can choose which of those to refer to / target. It would be bad practice to introduce a scope which is not part of the guidelines as only a single color scheme would use it.

Reply:
Okay... I don't mind going with the simple, ansi-specific approach of the first commit either. Let's see what other maintainers think.

Reply:
@sharkdp @Enselic any ideas on this one? we could use some input here. Thanks!

Reply:
I honestly don't know much about how to best and most properly structure themes and syntaxes. I fully trust Keiths judgement in these matters. He is one of the world's most prominent experts in this area :)

Reply:
@keith-hall I'm happy to go with the first commit approach. I also trust you know better about the usage of the themes and their scopes, etc., and if that doesn't look right to you, let's avoid it.